### PR TITLE
Add README.

### PR DESCRIPTION
**Pull request by ggwpez**

### CODE REVIEW

**Comment by ggwpez** [comment]
The import path is not yet working, but you can still take a look.  
You can obviously also make additions if you want. @matthiasgeihs

File: README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<h1 align="center"><br>
+    <a href="https://perun.network/"><img src=".assets/logo.png" alt="Perun" width="196"></a>
+<br></h1>
+
+<h4 align="center">Perun Ethereum Payment Client</h4>
+
+<p align="center">
+  <a href="https://www.apache.org/licenses/LICENSE-2.0.txt"><img src="https://img.shields.io/badge/license-Apache%202-blue" alt="License: Apache 2.0"></a>
+  <a href="https://pkg.go.dev/perun.network/go-perun?status.svg"> <img src="https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white" alt="pkg.go.dev docs"></a>
```
Comment (ggwpez) [inline README.md:9]: I will fix this once @tinnendo got the path running.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+</p>
+
+This project provides a simplified interfaec of the [*go-perun*](https://github.com/hyperledger-labs/go-perun) framework. It enables developers to get started quickly with *go-perun* by using a condensed API.
```
Comment (matthiasgeihs) [inline README.md:12]: typo: interfaec -> interface

File: README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+<h4 align="center">Perun Ethereum Payment Client</h4>
+
+<p align="center">
+  <a href="https://www.apache.org/licenses/LICENSE-2.0.txt"><img src="https://img.shields.io/badge/license-Apache%202-blue" alt="License: Apache 2.0"></a>
+  <a href="https://pkg.go.dev/perun.network/go-perun?status.svg"> <img src="https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white" alt="pkg.go.dev docs"></a>
+</p>
+
+This project provides a simplified interfaec of the [*go-perun*](https://github.com/hyperledger-labs/go-perun) framework. It enables developers to get started quickly with *go-perun* by using a condensed API.  
+If you just want to create a *go-perun* payment on Ethereum, this is for you.
```
Comment (matthiasgeihs) [inline README.md:13]: Create a go-perun payment? You mean create a payment application?

**Review by matthiasgeihs — CHANGES_REQUESTED** [review]
Minor corrections wrt language

File: README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+<br></h1>
+
+<h4 align="center">Perun Ethereum Payment Client</h4>
+
+<p align="center">
+  <a href="https://www.apache.org/licenses/LICENSE-2.0.txt"><img src="https://img.shields.io/badge/license-Apache%202-blue" alt="License: Apache 2.0"></a>
+  <a href="https://pkg.go.dev/perun.network/go-perun?status.svg"> <img src="https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white" alt="pkg.go.dev docs"></a>
+</p>
+
+This project provides a simplified interface of the [*go-perun*](https://github.com/hyperledger-labs/go-perun) framework. It enables developers to get started quickly with *go-perun* by using a condensed API.
```
Comment (matthiasgeihs) [inline README.md:12]: Interface "to"

**Review by matthiasgeihs — COMMENTED** [review]
I think we can use the github path. What do you think?

**Comment by ggwpez** [comment]
> I think we can use the github path. What do you think?

Yes right… Did not even think about this anymore :see_no_evil:  
I will remove the commit and change the `README` to just use `github.com/perun-network/perun-eth-payment`.

**Review by matthiasgeihs — APPROVED** [review]

**Pull request merged**